python_modules/dagster/dagster/core/execution/context/invocation.py

```diff
@@ -59,11 +59,13 @@ def __init__(
         resources_config: Dict[str, Any],
         instance: Optional[DagsterInstance],
         partition_key: Optional[str],
+        upstream_mapping_key: Optional[str],
```

Comment:
The fact that his is returned through `get_mapping_key`, not `get_upstream_mapping_key`, makes me think that this argument should just be called `mapping_key`. Thoughts?

Reply:
I personally think that's kind of misleading, but I guess consistency is probably more important. Perhaps the upstream part makes it clear.